src/compile/utils.ts

```diff
@@ -1,17 +1,18 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
 import fs from 'fs';
 import path from 'path';
 import yaml from 'yaml';
 
-import { head, isPlainObject, last, startCase } from 'lodash';
+import { isPlainObject, last, startCase } from 'lodash';
 
+// e.g. '#/definitions/flow/loop' => 'definitions.flow.loop'
```

Comment:
```suggestion

```

Reply:
why should it be deleted?

Reply:
my bad, I mistakenly took this for a duplicate